requirements_common.txt

```diff
@@ -14,13 +14,13 @@ pymarc==4.0.0; python_version >= '3.6'
 python-memcached==1.59
 simplejson==3.16.0
 web.py==0.39; python_version < '3.0'
-web.py==0.40; python_version >= '3.0'
+web.py==0.51; python_version >= '3.0'
```

Comment:
Was it intentional to update the version of web.py in this unrelated PR?

Reply:
Yes.